resources/hg-Encounter.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
De concepten Contactverantwoordelijke (804) en ContactMet(113) dataset lijkt op het zib2024 profiel Contact waarbij de rollen van de zorgverleners gesplitst zijn. Ik denk dat het goed is om een extra slice onder Encounter.participant te plaatsen voor ContactVerantwoordelijke, waarbij je de type vastzet. Dit ook omdat de kardinaliteit van ContactVerantwoordelijke 1..1 is en je hem hier in dit profiel nu niet verplicht stelt.

Reply:
Dank voor het meedenken en voor het voorbeeld van zib2024 (jammer dat we weer tegen een beperking aanlopen). Ik heb dit toegevoegd aan het profiel.